client/src/pages/Create.js

```diff
@@ -29,35 +32,56 @@ const Location = styled.div`
   }
 `;
 
+const CancelButton = styled(Button)`
+  color: #ffffff;
+  background: #de3a3a;
+`;
+
+const FooterGradient = styled.div`
+  height: 0.4em;
+  background: var(--bg-main-gradient);
+`;
+
+const Footer = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100vw;
+  background: #fff;
+  & button {
+    margin: 0.5em;
+  }
+`;
 export function Create() {
+  const [value, setValue] = useState("misc");
+  const [date, setDate] = useState(null);
+  // const [time, setTime] = useState(new Date());
+  const [time, setTime] = useState();
   const { register, handleSubmit, errors } = useForm();
```

Comment:
You have a mix of `useForm` states (see usage of `register`) and custom states like `date` or `time`.
It makes it difficult to maintain if you mix it :).